막대기.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const input = fs.readFileSync('/dev/stdin').toString().split('\n').map(v => +v);
```

Comment:
한줄에 80자 이상 넘기지 말기
화살표 함수 파라미터 항상 괄호 치기[(arrow-parens)](https://eslint.org/docs/rules/arrow-parens)
```suggestion
const input = fs
  .readFileSync('/dev/stdin')
  .toString()
  .split('\n')
  .map((v) => +v);
```